.github/workflows/fulcio-rekor-kind.yaml

```diff
@@ -203,6 +205,30 @@ jobs:
       run: |
         cosign verify --rekor-url ${{ env.REKOR_URL }} --allow-insecure-registry ${{ env.demoimage }}
 
+    # Test with cosign in 'airgapped mode'
+    # Uncomment these once modified cosign goes in.
```

Comment:
do we have a commit or version we should look for?

Reply:
I'll grab the version after it gets committed. I tested with my own fork for functionality, but will need to update the instructions, documentation, so will do after it's been released. Is that ok?

Reply:
Sorry, I'll grab the 'release' once it gets released with my commit there.